## SLURM PROLOG ###############################################################
##    Job ID : 768965
##  Job Name : debug
##  Nodelist : gpu1505
##      CPUs : 1
##  Mem/Node : 16384 MB
## Directory : /oscar/data/jtompki1/yliang51/GaussianDiff/slurms
##   Job Started : Sat Feb 24 12:09:23 AM EST 2024
###############################################################################
conda 23.1.0
/users/yliang51/data/yliang51/envs/gaufre/bin/python
/users/yliang51/data/yliang51/envs/gaufre/bin/pip
[rank: 0] Global seed set to 3057170324
wandb: Currently logged in as: lynl7130. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.16.3
wandb: Run data is saved locally in ./output/base/wandb/run-20240224_000934-7zc8boa9
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run base
wandb: ‚≠êÔ∏è View project at https://wandb.ai/lynl7130/base
wandb: üöÄ View run at https://wandb.ai/lynl7130/base/runs/7zc8boa9
output path: ./output/base
0it [00:00, ?it/s]164it [00:00, 73781.60it/s]
GPU available: True (cuda), used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
wandb: - 0.006 MB of 0.006 MB uploadedwandb: \ 0.006 MB of 0.006 MB uploadedwandb: | 0.006 MB of 0.016 MB uploadedwandb: üöÄ View run base at: https://wandb.ai/lynl7130/base/runs/7zc8boa9
wandb: Ô∏è‚ö° View job at https://wandb.ai/lynl7130/base/jobs/QXJ0aWZhY3RDb2xsZWN0aW9uOjE0MjQ5NDQ0Nw==/version_details/v0
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./output/base/wandb/run-20240224_000934-7zc8boa9/logs
Traceback (most recent call last):
  File "main.py", line 10, in <module>
    subclass_mode_data=True
  File "/users/yliang51/data/yliang51/envs/gaufre/lib/python3.7/site-packages/lightning/pytorch/cli.py", line 358, in __init__
    self._run_subcommand(self.subcommand)
  File "/users/yliang51/data/yliang51/envs/gaufre/lib/python3.7/site-packages/lightning/pytorch/cli.py", line 670, in _run_subcommand
    fn(**fn_kwargs)
  File "/users/yliang51/data/yliang51/envs/gaufre/lib/python3.7/site-packages/lightning/pytorch/trainer/trainer.py", line 609, in fit
    self, self._fit_impl, model, train_dataloaders, val_dataloaders, datamodule, ckpt_path
  File "/users/yliang51/data/yliang51/envs/gaufre/lib/python3.7/site-packages/lightning/pytorch/trainer/call.py", line 38, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/users/yliang51/data/yliang51/envs/gaufre/lib/python3.7/site-packages/lightning/pytorch/trainer/trainer.py", line 650, in _fit_impl
    self._run(model, ckpt_path=self.ckpt_path)
  File "/users/yliang51/data/yliang51/envs/gaufre/lib/python3.7/site-packages/lightning/pytorch/trainer/trainer.py", line 1038, in _run
    verify_loop_configurations(self)
  File "/users/yliang51/data/yliang51/envs/gaufre/lib/python3.7/site-packages/lightning/pytorch/trainer/configuration_validator.py", line 41, in verify_loop_configurations
    __verify_train_val_loop_configuration(trainer, model)
  File "/users/yliang51/data/yliang51/envs/gaufre/lib/python3.7/site-packages/lightning/pytorch/trainer/configuration_validator.py", line 71, in __verify_train_val_loop_configuration
    "No `training_step()` method defined. Lightning `Trainer` expects as minimum a"
lightning.fabric.utilities.exceptions.MisconfigurationException: No `training_step()` method defined. Lightning `Trainer` expects as minimum a `training_step()`, `train_dataloader()` and `configure_optimizers()` to be defined.
